designing-reports/report-book/report-book-parameters.md

```diff
@@ -11,28 +11,86 @@ previous_url: /designing-reports-reportbook-report-parameters
+>note When one or more parameters are merged, the UI settings of the __first__ one are used for the generated input control in the parameters area.
 
 ![An image showing how the values of the Report Parameters of multiple Reports with same names are merged together when Mergeable is true](images/ReportBook3_MergedParameters.png)
 
 You can control parameter merging via the [`Mergeable`](/api/Telerik.Reporting.IReportParameter#Telerik_Reporting_IReportParameter_Mergeable) boolean property of the `ReportParameter` class. The default value of the `Mergeable` property is true. Set the `Mergeable` property to `false` if you want to prevent a parameter from being merged with its equivalent ones.
 
 ![An image showing how the values of the Report Parameters of multiple Reports with same names keep their individual values when Mergeable is false](images/ReportBook4_MergedParameters2.png)
 
-## Passing Values for Report Parameters in a Reportbook through ReportSource
+## How to set a value for a report parameter in ReportBook
+
+In the sample pictures below, you will see in few steps how to set a value for a report parameter in ReportBook using Standalone Designer. For this example, the **Dashboard.trdp** and **Product Sales.trpd** reports are used, which can be found in the installation folder - **C:\Program Files (x86)\Progress\Telerik Reporting 2025 Q1\Report Designer\Examples**.
+
+### In Standalone Designer
+
+1. Click on the **Dashboard.trdp**
+1. From the Toolbar, click on the **Parameters**
+1. The **Edit Parameter** window will open, then select New
+
+![How to set a value for a report parameter using Standalone Designer](images/SetValueForReportParameterInReportBookStandaloneDesigner.png)
+
+### Web Report Designer
+
+1. In Report Sources, click on the `uriReportSources1 `
+1. Next to Parameters, you will see a plus sign(+). Click on it.
+1. In the `Add New Item` window, you will be able to set the name and value of the parameter.
+
+![How to set a value for a report parameter using Web Report Designer](images/HowToSetValueForReportParameterWebReportDesigner.png)
+
+![How to set the name and value of the parameter](images/HowToSetTheNameAndValueOfTheParameter.png)
+
+## Passing Values for Report Parameters in a Reportbook through ReportSource Programatically
 
 * __When report parameters have unique names or the target parameters are merged:__ If a parameter is distinguishable by its Name property or the Name denotes several merged parameters, refer to the parameter directly by the value of its Name property.
 
-{{source=CodeSnippets\CS\API\Telerik\Reporting\ReportSourceSnippets.cs region=Set_Values_For_Unique_Or_Mergable_ReportParameters_In_ReportSource_Snippet}}
-{{source=CodeSnippets\VB\API\Telerik\Reporting\ReportSourceSnippets.vb region=Set_Values_For_Unique_Or_Mergable_ReportParameters_In_ReportSource_Snippet}}
+````C#
```

Comment:
I think it is much better to continue referencing the code from the CodeSnippets in the Reporting solution. Just fix the code there if necessary.
To avoid duplicating variables, use different scopes and regions. For example:

int a = 0;
if (a == 0)
{ 
#region 1
// code snippet 1 here
#endregion
} else if (a == 1)
{
#region 2
// code snippet 2 here
#endregion
}
etc.